Add `fill` option

medusalix opened the pull request:
I added support for the [`fillColor`](https://developer.apple.com/documentation/appkit/nsworkspace/desktopimageoptionkey/1535407-fillcolor) property to the API and made it accessible through the CLI-option `--fill`. Setting the background color without an image URL is not possible, though I looked into the way the system prefpane allows you to choose a custom color. Apparently it stores a transparent picture in the bundles' resources: `/System/Library/PreferencePanes/DesktopScreenEffectsPref.prefPane/Contents/Resources/DesktopPictures.prefPane/Contents/Resources/Transparent.tiff`. It might be possible to use this image to add the additional functionality of setting a solid color background to the CLI.

Resolves #19

medusalix commented:
@sindresorhus Any thoughts?

sindresorhus commented on readme.md:
```diff
@@ -99,7 +100,7 @@ import Wallpaper
 let imageURL = URL(fileURLWithPath: "<path>")
 
-try! Wallpaper.set(imageURL, screen: .main, scale: .fill)
+try! Wallpaper.set(imageURL, screen: .main, scale: .fill, fill: "FF0000")
```
This is not correct. It accepts a NSColor.

sindresorhus commented on readme.md:
```diff
@@ -65,6 +65,7 @@ Set wallpaper image
 Options:
   --scale <value>     Values: auto, fill, fit, stretch, center [Default: auto]
   --screen <value>    Values: all, main, <index> [Default: all]
+  --fill <value>      Format: Hex color <RRGGBB> [Default: nil]
```
I think `--fill-color` would be clearer`.

sindresorhus commented:
This looks good for setting a fill color with an image, but there needs to be a way to set just a color too as solid background, per the #19. We can use the transparent image trick.

For the API, it could just be an overload of `set`:

```swift
try Wallpaper.set(
	NSColor.systemRed,
	screen: convertStringToScreen(screen.value)
)
```

(Without `scale` and `fillColor` parameters)

As for the CLI, I'm not sure how to model this. Suggestions welcome.

sindresorhus commented:
@medusalix Should I close this?

medusalix commented:
@medusalix I'm still interested in finishing this (as soon as I'm back on my macOS machine).

medusalix commented:
@sindresorhus I've corrected the usage in the README and renamed the flag to `--fill-color`. I've started working on the solid color functionality, but this PR needs to be merged first.
